packages/babel-preset-env/src/index.ts

```diff
@@ -160,15 +157,12 @@ export const getModulesPluginNames = ({
     if (!shouldTransformDynamicImport) {
       modulesPluginNames.push("syntax-dynamic-import");
     }
-    if (!shouldTransformExportNamespaceFrom) {
-      modulesPluginNames.push("syntax-export-namespace-from");
```

Comment:
This does not need to be included, because this plugin also goes through the normal process for including transform/syntax plugins. The only reason this plugin has special handling is that we must forcefully enable the transform if the targets support it but the caller does not.

Maybe we should get rid of this behavior in Babel 8, similarly to how the caller does not affect any other plugin.